jasmine-standalone-3.4.0/spec/scoreSpec.js

```diff
@@ -0,0 +1,43 @@
+
+
+describe('Bowling game', function(){
+
+  var game;
+
+  beforeEach(function(){
+    game = new BowlingGame();
+  });
+
+  it("can roll a gutter game", function() {
+    rollMany(0,20);
+    expect(game.score()).toBe(0);
+  });
+
+  it("can roll all ones", function() {
+    rollMany(1,20);
+    expect(game.score()).toBe(20);
+  });
+
+  it("can roll a spare", function() {
+      game.roll(5);
+      game.roll(5);
+      game.roll(3);
+      rollMany(0,17);
+      expect(game.score()).toBe(16);
+  });
+
+  it("can roll a strike", function() {
```

Comment:
I would change the it here to be - can roll a single strike.
There is another use case where you can roll multiple strikes in a row. However - I may being premature here as you function might cover all these.